src/style.css

```diff
@@ -0,0 +1,112 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 600;
+}
+
+:root {
+  --color-border: #e1e7ed;
+  --color-input: #3d4e61;
+}
+
+label {
+  display: flex;
+  width: 100%;
+}
+
+.big__search {
+  box-sizing: border-box;
```

Comment:
Move to the  * selector and remove it from all other places.

Reply:
when I write * to all selectors, the search images disappear

Reply:
Feel free to ask for help in the chat